api/V1/recordingUtil.ts

```diff
@@ -1098,6 +1106,37 @@ function addAudioBaitRow(out: any, audioBait: Event) {
     ""
   ]);
 }
+
+async function sendAlerts(recID: number) {
+  const recording = (await models.Recording.getForAdmin(recID)) as any;
+  const recVisit = new Visit(recording, 0);
+  recVisit.completeVisit();
+  let matchedTrack, matchedTag;
+  // find any ai master tags that match the visit tag
+  for (const track of recording.Tracks) {
+    matchedTag = track.TrackTags.find(
+      (tag) => tag.data == AI_MASTER && recVisit.what == tag.what
+    );
+    if (matchedTag) {
+      matchedTrack = track;
+      break;
+    }
+  }
+  if (!matchedTag) {
+    return;
+  }
+
+  const alerts = await (models.Alert as AlertStatic).getActiveAlerts(
```

Comment:
It seems like it would be much less work to check for active alerts at the top of this function, and exit early if there are no alerts?